tests/cli/samples/logs.yml

```diff
@@ -1,4 +1,5 @@
 - logs-all:
+  run: true
```

Comment:
can we have a default value for run (true)? this way we only have to specify this key when we want to disable a test.